tools/pg_inspector/schedule_server_dump.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+basedir=$(dirname $0)
+absdir=$(realpath $basedir)
+
+crontab -l 2>/dev/null | grep -v pg_inspector >crontab.tmp
+echo "PATH=${PATH} # pg_inspector path to find ruby" >>crontab.tmp
+echo "0 0 * * * ${absdir}/dump_server.sh" >>crontab.tmp
+crontab crontab.tmp
+rm -f crontab.tmp
```

Comment:
This script should live in next to this file:  https://github.com/ManageIQ/manageiq-appliance/blob/master/LINK/etc/cron.hourly/miqtimestamp-hourly.cron (and call the above renamed and moved file).  When you're in the office, ping one of us to show you how to test this.

Reply:
Thanks, will move it